packages/paste-theme-tokens/package.json

```diff
@@ -13,7 +13,7 @@
     "dist"
   ],
   "scripts": {
-    "build": "yarn clean && yarn compile",
```

Comment:
This one doesn't need a clean?

Reply:
Good catch.  I think this is an artifact from when I was debugging the js files being built in the src folder.